### PR TITLE
cri-o: error out when node is a Docker container

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1489555

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: roles/docker/tasks/systemcontainer_crio.yml [hunk 1]
```diff
@@ -36,6 +36,12 @@
     state: present
   when: not openshift.common.is_atomic | bool
 
+- name: Check we are not using node as a Docker system container with CRI-O
```
Comment (ashcrow) [inline roles/docker/tasks/systemcontainer_crio.yml]: Shouldn't this be for Docker container with the system container being OK?

Comment (giuseppe) [reply roles/docker/tasks/systemcontainer_crio.yml]: ouch, yes!   Pushed a new version

**Comment by ashcrow** [comment]
```
No matching distribution found for ansible==2.3.1.0 (from -r requirements.txt (line 3))
```

**Comment by ashcrow** [comment]
bot, retest this please

**Pull request merged**